src/core/BufferAttribute.js

```diff
@@ -257,6 +257,12 @@ class BufferAttribute {
 
 	getZ( index ) {
 
+		if ( this.itemSize < 3 ) {
+
+			return 0;
+
+		}
```

Comment:
It think we should also update `setZ()` in a similar fashion for consistency reasons.